src/ProjectFileManager.cpp

```diff
@@ -1306,6 +1308,26 @@ class ImportProgress final
 
    }
 
+   void OnImportStarted(const wxString &filename) override
+   {
+       mImportInfoDialog = new ImportInfoDialog(filename, NULL, -1, XO("Importing files"));
```

Comment:
This dialog will appear on every import (even for tiny files), won't it be too intrusive?

Reply:
I don't think there is a way to determine if file is already readable or in the middle of download so I guess it needs to stay that way.
There's other option though: there is a progress dialog displayed right after obtaining file handle - instead of showing ImportInfo dialog and then ProgressDialog, we could display only ProgressDialog but sooner - but this needs some refactoring because ProgressDialog expects file handle in its constructor.
Right now it looks like ImportInfo dialog "evolves" into ProgressDialog